.travis.yml

```diff
@@ -1,4 +1,20 @@
-language: java
+language: android
 jdk:
   - openjdk7
   - oraclejdk8
+env:
+  - TEST=java
+  - TEST=android
+matrix:
+  exclude:
+    - jdk: oraclejdk8
+      env: TEST=android
+android:
+  components:
+    - android-15
+    - build-tools-21.1.1
```

Comment:
I think I tried it, but it didn't work. Or maybe I was confused because it wasn't on the list here: https://docs.travis-ci.com/user/languages/android#Pre-installed-components

I'll try updating the build-tools and sys-img, maybe it works now.